charts/shoot-addons/charts/kubernetes-dashboard/templates/deployment-dashboard.yaml

```diff
@@ -75,3 +69,6 @@ spec:
           secretName: kubernetes-dashboard-certs
       - name: tmp-volume
         emptyDir: {}
+      serviceAccountName: kubernetes-dashboard
+      nodeSelector:
+        "beta.kubernetes.io/os": linux
```

Comment:
This label is deprecated since `1.14` can you create function that returns `kubernetes.io/os` for  versions >= `1.14`?

Reply:
Nice catch, thanks! Actually, I took this from https://github.com/kubernetes/dashboard/blob/v2.0.0-beta8/aio/deploy/recommended/06_dashboard-deployment.yaml#L74-L75, but I don't see too much value in it right now. I completely removed the `nodeSelector`. I assume if somebody would run a non-linux worker node we would have much bigger problems and not just a not-running kubernetes-dashboard ;-)